modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx

```diff
@@ -139,9 +139,14 @@ export default function EditNotificationTemplate({
 
 			window.location.assign(document.referrer);
 		}
-		else if (response.status === 404) {
```

Comment:
Can you guarantee that a 404 will never happen? I wouldn't change this case. I'd leave it as it is and create a whole new `else if`, before this one, to handle the 400 case.